backend/clubs/serializers.py

```diff
@@ -1169,6 +1211,20 @@ def validate_student_types(self, value):
         Ensure that the user does not select all of the student types.
         """
         if len(value) >= StudentType.objects.count():
+            if settings.BRANDING == "fyh":
+                raise serializers.ValidationError(
+                    "We noticed you selected all of the options\
+                     for one or more of the previous tags."
+                    "In order to best optimize our sorting algorithm,\
+                     you need to select only the few \
+                    tags that apply to your resource."
+                    "If you feel that all the tags apply, that's great!\
+                     In that case you would select 'Yes' to the \
+                    question asking if your resource applies to\
+                     all undergraduate, graduate, and professional Penn students."
+                    "Thanks for doing your part to ensure that Hub@Penn\
+                     quickly and efficiently gets resources to our Penn community. "
```

Comment:
This is a ton of code duplication.

Consider saving this string to a variable and using the variable throughout the program?

Reply:
Imagine if they wanted to change this text, it would be easy to miss an entry if you're not aware that there are multiple.